CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 # CMake settings
 #===============================================================================
 cmake_minimum_required(VERSION 2.8.6 FATAL_ERROR)
+cmake_policy(SET CMP0042 NEW)
```

Comment:
[It seems cmake less than 3.0.0 doesn't recognize this policy](https://travis-ci.org/dartsim/dart/builds/74006048). There is policy existence checker which is supported by cmake 2.8.6. How about this?

```cmake
if(POLICY CMP0042)
  # MACOSX_RPATH is enabled by default.
  cmake_policy(SET CMP0042 NEW)
endif(POLICY CMP0042)
```